Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/05-PersistingSyncUps/PersistingSyncUps-02-code-0001.swift

```diff
@@ -33,12 +33,26 @@ struct SyncUpsListTests {
 
     await store.send(.confirmAddButtonTapped) {
       $0.addSyncUp = nil
-      // $0.syncUps = [editedSyncUp]
+      $0.syncUps = [editedSyncUp]
     }
   }
 
   @Test
   func deletion() async {
-    // ...
+    @Shared(.syncUps) var syncUps = [
+      SyncUp(
+        id: SyncUp.ID(),
+        title: "Point-Free Morning Sync"
+      )
+    ]
+    let store = TestStore(
+      initialState: SyncUpsList.State()
+    ) {
+      SyncUpsList()
+    }
```

Comment:
📓 First we needed to update the deletion test to update it's initialization with the setup of the `@Shared` outside the `TestStore`